### DISCUSSION TITLE
Gruvbox theme

### DISCUSSION THREAD

**Discussion by Mohammadreza99A** [tsujan/Kvantum#622] [Q&A]
Is it possible to have a gruvbox theme?

**Comment by tsujan** [comment]
I found it by googling and didn't like it. So, I won't add it to the themes that come with Kvantum's installation but theme makers could make it for Kvantum. In that sense, yes, it's possible, of course.

**Reply by Mohammadreza99A** [reply]
Is there an app that lets to create kvantum theme? Or I should do it by creating a .kconfig file?

**Reply by tsujan** [reply]
Everything is explained in https://github.com/tsujan/Kvantum/tree/master/Kvantum/doc.

No, there isn't an app for making a Kvantum theme automatically. Theme making isn't easy and needs patience and practice — as it does in the case of GTK themes.

For a theme as simple as Gruvbox, you might want to use Breeze with a suitable color scheme instead.
